src/views/guestLayout.js

```diff
@@ -42,28 +42,32 @@ const guestLayout = data => `
 
     <!-- Google tag (gtag.js) -->
     <script nonce='${data.nonce}'>
-      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtag/js?id='+i+dl;var n=d.querySelector('[nonce]');
-      n&&j.setAttribute('nonce',n.nonce||n.getAttribute('nonce'));f.parentNode.insertBefore(j,f);
-      })(window,document,'script','dataLayer','${AppConstants.GA4_MEASUREMENT_ID}');
-      function gtag(){dataLayer.push(arguments);}
-      ${AppConstants.GA4_DEBUG_MODE
-        ? `gtag('config', '${AppConstants.GA4_MEASUREMENT_ID}', { 'debug_mode': true })`
-        : ''}
-      gtag('js', new Date());
-      gtag('config', '${AppConstants.GA4_MEASUREMENT_ID}',
-        { cookie_domain: window.location.hostname, cookie_flags: "SameSite=None;Secure" }
-      );
-      window.gtag = gtag
+      if (!navigator.doNotTrack || navigator.doNotTrack !== '1') {
```

Comment:
Shouldn't `navigator.doNotTrack !== '1'` also cover the case when navigator.doNotTrack is undefined?